chore: update documentation in README.md (#20)
vaadin/eslint-config-vaadin: README.md

## Usage

After you have installed `eslint-config-vaadin`, you should be able to extend the following configs in the [`extends`](http://eslint.org/docs/user-guide/configuring#extending-configuration-files) section of your [ESLint configuration](http://eslint.org/docs/user-guide/configuring).

## Available Configs

### eslint-config-vaadin

> A config config that contains the JavaScript linting rules.

Author: ```suggestion
The default config that contains the JavaScript linting rules.
```

This config requires `eslint-config-vaadin/prettier` that must be added the very last i.e. after any other configs.

```json
{
  "extends": [
    "eslint-config-vaadin",
    // ... any other configs
    "eslint-config-vaadin/prettier"
  ]
}
```

### eslint-config-vaadin/typescript

A config that contains the TypeScript linting rules.

> This config is designed on top of the default `eslint-config-vaadin` config which means that you don't have to enable the default config when using this one.

Author: ```suggestion
This config is designed on top of the default `eslint-config-vaadin` config so you don't have to enable the default config when using this one.
```
